graphene/types/resolver.py

```diff
@@ -6,7 +6,14 @@ def dict_resolver(attname, default_value, root, info, **args):
     return root.get(attname, default_value)
 
 
-default_resolver = attr_resolver
+def dict_or_attr_resolver(attname, default_value, root, info, **args):
+    resolver = attr_resolver
+    if isinstance(root, dict):
```

Comment:
`resolver = dict_resolver if isinstance(root, dict) else attr_resolver` is more Pythonic IMHO